regression/test_config.yaml

```diff
@@ -88,7 +88,7 @@ PascalVOC_2007_2012_256_320_augmented:
     threshold: -500
     epoch: 15
 Kinetics400:
-  datapath: "/data_ssd"
+  datapath: "/data/alican_data/"
```

Comment:
Use a generic destination and update documentation

Reply:
It is fixed.